src/array/list.rs

```diff
@@ -33,9 +32,9 @@ where
             opt.map(|array| {
                 array
                     .as_any()
-                    .downcast_ref::<PrimitiveArray<u64>>()
+                    .downcast_ref::<UInt64Array>()
```

Comment:
btw you can use [`as_primitive`](https://docs.rs/arrow/latest/arrow/array/trait.AsArray.html#method.as_primitive) if you want to save a few lines.

Reply:
Nice - thanks